zone/bot.cpp

```diff
@@ -6411,21 +5747,21 @@ bool Bot::SpellOnTarget(
 
 bool Bot::IsImmuneToSpell(uint16 spell_id, Mob *caster) {
 	bool Result = false;
-	if(!caster)
+	if (!caster)
 		return false;
 
-	if(!IsSacrificeSpell(spell_id) && !(zone->GetZoneID() == 202) && !(this == caster)) {
+	if (!IsSacrificeSpell(spell_id) && zone->GetZoneID() != Zones::POKNOWLEDGE && this != caster) {
```

Comment:
Got to be a better way to handle the zone logic here. Seems this is unrelated to raids though